jsk_baxter_robot/jsk_baxter_startup/baxter.launch

```diff
@@ -20,8 +21,22 @@
   <node if="$(arg launch_servo)" pkg="baxter_tools" name="baxter_servo_node" type="enable_robot.py" args="-e" output="screen" clear_params="true"/>
 
   <!-- Startup joint trajectory will enable send command from eus-->
-  <node if="$(arg launch_joint_trajectory)" pkg="baxter_interface" name="baxter_joint_trajectory" type="joint_trajectory_action_server.py" args="--interpolation minjerk" output="screen"/>
-  <node if="$(arg launch_joint_trajectory)" pkg="baxter_interface" name="head_joint_trajectory" type="head_action_server.py" args="" output="screen"/>
+  <group if="$(arg launch_joint_trajectory)">
+    <node name="baxter_joint_trajectory"
+          pkg="baxter_interface" type="joint_trajectory_action_server.py"
+          args="--interpolation minjerk" output="screen"/>
```

Comment:
note that minjerk is not released yet, it's only exists  in development branchhttps://github.com/RethinkRobotics/baxter_interface/pull/78/files